full-service/Cargo.toml

```diff
@@ -47,6 +48,9 @@ mc-validator-connection = { path = "../validator/connection" }
 mc-watcher = { path = "../mobilecoin/watcher" }
 mc-watcher-api = { path = "../mobilecoin/watcher/api" }
 
+ledger-mob = { path = "../ledger-mob/lib" }
```

Comment:
Optional/Nit: Move this above the `mc-*` deps so that relative path imports are sorted